File: riesi.java
import java.util.Scanner;
public class riesi {
	
	public static void main (String[] args) {
		System.out.println("Riesi: False");
		System.out.println("Riesi?");
		Scanner in = new Scanner(System.in);
		char inp = in.next().charAt(0);
		if(inp == 'y'){
			System.out.println("Riesi: True");
		}else if(inp == 'n'){
			System.out.println("Riesi: False");
		}
	}
}

File: riesi.go
package main 
  
import (
 "fmt"
)

func main() { 
    var riesi string
    fmt.Println("Riesi: False")
    fmt.Println("Riesi? ")
    fmt.Scanln(&riesi)

    if riesi == "y" {
      fmt.Print("Riesi: True")
    } else {
      fmt.Println("Riesi: False")
    }
}
